common/shlibs

```diff
@@ -4018,3 +4018,4 @@ libXcompshad.so.3 nx-libs-3.5.99.24_1
 libNX_X11.so.6 nx-libs-3.5.99.24_1
 librnnoise.so.0 rnnoise-0.4.1_1
 liblhasa.so.0 lhasa-0.3.1_2
+sambausershareplugin.so kdenetwork-filesharing-21.04.3_1
```

Comment:
Is this actually necessary? If it's just to silence a warning, please don't add it.